resources/v2/01_students.sql

```diff
@@ -2,8 +2,6 @@ CREATE TABLE students
 (
     id SERIAL NOT NULL PRIMARY KEY,
     name text NOT NULL,
-    city text NOT NULL,
-    age integer NOT NULL,
```

Comment:
Why have you removed this and created new resource where you do the same ?

Reply:
Since `students` table was already migrated, hence to update the update the `students` table with new columns, we have to create new `resources` with new migrations.

Or, I could manually delete the `tables` from `TablePlus` and add the desired fields in the same file which is not the correct way to approach.